lecture/forms.py

```
@@ -0,0 +1,20 @@
+from django import forms
+from .models import Lecture, LecturePDF
+
+class LectureForm(forms.ModelForm):
+    class Meta:
+        model = Lecture
+        fields = ['title', 'classroom']
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)